bin/describe/describe_workspace.ml

```diff
@@ -72,9 +72,9 @@ module Descr = struct
   module Mod_deps = struct
     type t =
       { for_intf : Dune_rules.Module_name.t list
-          (* direct module dependencies for the interface *)
+            (* direct module dependencies for the interface *)
```

Comment:
This seems to be indented too bar. The original is already misindented but this makes it worse.

Reply:
Excellent, glad to hear. The new aligning is very neat, I'm looking forward to the final release.